### DISCUSSION TITLE
Set different waves for different instruments

### DISCUSSION THREAD

**Discussion by cheburashka777** [squiggythings/WaveTracker#26] [General]
How can I set different waves for different instruments? The documentation doesn't say anything about it, and there's no such button in the interface. So Wave Bank is not needed, because all instruments use the same wave for sound.

**Comment by squiggythings** [comment]
Waves can be set per channel with the Vxx effect, this allows you to use the same instrument but with different waves. If you want an instrument to always use a certain wave, regardless of what was set with the Vxx effect, you can add a 'Wave' envelope to the instrument, which lets you set which wave the instrument uses over time.
